### PR TITLE
feat: turn playing info to a component

### PR DESCRIPTION
**Pull request by yuzh0816**
Also beautify login background(backdrop blur) and updated turbo.

### CODE REVIEW

File: packages/web/components/Topbar/SearchBox.tsx [hunk 1]
```diff
@@ -160,12 +160,12 @@ const SearchBox = () => {
           `
         )}
       >
-        <SearchIcon />
+        <SearchIcon isSearching={false} />
         <input
           ref={inputRef}
-          placeholder={t`search.search`}
+          placeholder={t`search.search`.toString()}
```
Comment (pan93412) [inline packages/web/components/Topbar/SearchBox.tsx:166]: Is this change necessary?

Comment (yuzh0816) [reply packages/web/components/Topbar/SearchBox.tsx:166]: It's detected by vscode, so I think it's necessary to fix that.

**Review by pan93412 — APPROVED** [review]
